fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -1804,7 +1804,8 @@ public Range getUnbuiltRange() {
                                     LogMessageKeys.INDEX_NAME, index.getName(),
                                     "unbuiltRangeBegin", ByteArrayUtil2.loggable(firstUnbuilt.get().begin),
                                     "unbuiltRangeEnd", ByteArrayUtil2.loggable(firstUnbuilt.get().end),
-                                    subspaceProvider.logKey(), subspaceProvider);
+                                    subspaceProvider.logKey(), subspaceProvider,
+                                    "indexSubspace", indexSubspace(index));
```

Comment:
This will be a suffix of the subspace provider's resolved path, but without the symbols. We could just log the index's subspace key. Or do you think this makes it clearer?

Reply:
Done.